.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We should exclude this file from the PR

Reply:
Overall good stuff. 
 
I have comments and questions:
 

	Python quickstart is coming along.  I like the changes.  
	Where is the code for these?  I think the example is still backwards where an order processor calls the cart.  
	.NET needs a bit of work.  
	
		The term we should use is simply .NET SDK or .NET 6 SDK.  "Core" is not term we use anymore.  SDK is the specific deliverable vs. the runtime.  the runtime isn't enough.  
		The hard coded download link isn't what we want.  lets use this: https://dotnet.microsoft.com/en-us/download
		I don't understand what we mean by get latest Nuget packages a prereq.  We can remove that.  Instead of doing that, have a step to `dotnet restore` and `dotnet build `  These will bring the nuget package refs back for you, assuming they are already in sample.  This needs to happen before `dapr run` steps.  This is moral equivalent of the `pip install` you did in Python.  Dotnet is a compiled language so that dotnet build also needs to happen, not just the restore.  
		The code is inside a `main` function loop.  In .NET 6 we don't do that anymore.  You just start coding in the Program.cs without a main.  See this example: [ASPNETCoreFrontEndWithBackendAPIsonContainerApps/Program.cs at paulyuk/dapr · paulyuk/ASPNETCoreFrontEndWithBackendAPIsonContainerApps (github.com)](https://github.com/paulyuk/ASPNETCoreFrontEndWithBackendAPIsonContainerApps/blob/paulyuk/dapr/DotNetOnContainerApps.FrontEnd/Program.cs)
		Also for the snip of code we just want to show the publish or subscribe part, not all the imports and sleeps, etc.
	
	
	Javascript
	
		instead of npm install on daprclient, should we just `npm install` and let the package list take care of it?
		do we need an npm run buildclient ?
		should we do apples to apples with python and remove the function wrapping the publish code snippet?
	
	
	deploy to cloud -- in a future rev (dont block on it now) i'd like to point to a cloud integrations page that show how to deploy to each cloud.  We could write the azure one and invite community to do the others